src/components/ExploreMore.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+import React from "react";
+import { Box, Button, Typography } from "@mui/material";
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import TitleResponse from "./TitleResponse";
+
+interface ExploreMoreData {
+	headline: string;
+	startupName: string;
+	descriptionShort: string;
+	productLink: string;
+	demoLink: string;
+	descriptionLong: string;
```

Comment:
The layout for displaying startup details looks clear and actually very easy to read, and including clickable links for productLink and demoLink is a nice touch. Maybe just add handling for missing links to avoid broken URLs.

Reply:
That's a great point on the edge cases. I added a check now